Modules/Core/Transform/include/itkBSplineTransform.hxx

```diff
@@ -440,8 +440,7 @@ BSplineTransform<TParametersValueType, VDimension, VSplineOrder>::SetCoefficient
 
   if (!validArrayOfImages)
   {
-    itkExceptionMacro("SetCoefficientImage() requires that an array of "
-                      << "correctly sized images be supplied.");
+    itkExceptionMacro("SetCoefficientImage() requires that an array of " << "correctly sized images be supplied.");
```

Comment:
`" << "` in the middle of the string can be removed now. Is there an automated way to accomplish that?

Reply:
Done.

Reply:
Removing " << " is a real code change. It's not just a format change. Although it's a change that I like, I would prefer such a change in a separate PR.

Reply:
Follow-up in PR #5052.